dbt/include/duckdb/macros/materializations/external.sql

```diff
@@ -89,6 +96,6 @@
 
   {{ run_hooks(post_hooks, inside_transaction=False) }}
 
-  {{ return({'relations': [target_relation]}) }}
+  {{ return({'relations': [target_relation]}) }} -- to what i return?
```

Comment:
This is returning a `Relation` object in the dbt API: https://github.com/dbt-labs/dbt-adapters/blob/main/dbt/adapters/base/relation.py#L40